install/utils/class.System.php

```
@@ -156,12 +156,12 @@ public static function getAvailableLocales($path)
                             $expectedUri = 'http://www.tao.lu/Ontologies/TAO.rdf#Lang' . $l;
 
                             // Look for an rdf:value equals to the folder name.
-                            $rdfValues = $xpath->query("//rdf:Description[@rdf:about='${expectedUri}']/rdf:value");
+                            $rdfValues = $xpath->query("//rdf:Description[@rdf:about='{$expectedUri}']/rdf:value");
                             if ($rdfValues->length == 1 && $rdfValues->item(0)->nodeValue == $l) {
                                 $key = $l;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$key'. (undefined)

(UnusedLocalVariable)